packages/core/hooks/useOpenWithMenuItems/index.tsx

```diff
@@ -299,6 +347,65 @@ export default (fileDetails?: FileDetail, filters?: FileFilter[]): IContextualMe
         fileService
     );
 
+    // custom hook this, like `useOpenInCfe`?
+    const openInVole = React.useCallback(async (): Promise<void> => {
+        const allDetails = await fileSelection.fetchAllDetails();
+        const details = allDetails.filter((detail) => {
+            const fileExt = getFileExtension(detail);
+            return fileExt === "zarr" || fileExt === "";
```

Comment:
Sldy files are sometimes captured in the community as `.sldy/` which would become `""` here

Reply:
`getFileExtension` is unaware of `/` and would get this right.

...which implies a bigger problem: this will fall over for valid Zarr images without an extension with a `.` somewhere else in their file path: `.../john.doe/valid_zarr`. I'll look at making that more robust, including special-casing `.sldy/`.